code/modules/roguetown/roguecrafting/alchemy/reagents.dm

```diff
@@ -296,24 +292,24 @@ If you want to expand on poisons theres tons of fun effects TG chemistry has tha
 	reagent_state = LIQUID
 	color = "#083b1c"
 	taste_description = "breathlessness"
-	metabolization_rate = REAGENTS_SLOW_METABOLISM
+	metabolization_rate = REAGENTS_SLOW_METABOLISM * 3
 
 /datum/reagent/stampoison/on_mob_life(mob/living/carbon/M)
 	if(!HAS_TRAIT(M,TRAIT_NOSTAMINA))
-		M.adjust_stamina(-25) //Slowly leech stamina
+		M.adjust_stamina(2.25) //Slowly leech stamina
 	return ..()
 
-/datum/reagent/strongstampoisons
+/datum/reagent/strongstampoison
 	name = "Strong Stamina Poison"
 	description = ""
 	reagent_state = LIQUID
 	color = "#041d0e"
 	taste_description = "frozen air"
-	metabolization_rate = REAGENTS_SLOW_METABOLISM * 3
+	metabolization_rate = REAGENTS_SLOW_METABOLISM * 9
 
 /datum/reagent/strongstampoison/on_mob_life(mob/living/carbon/M)
 	if(!HAS_TRAIT(M,TRAIT_NOSTAMINA))
-		M.adjust_stamina(-150) //Rapidly leech stamina
+		M.adjust_stamina(9) //Rapidly leech stamina
```

Comment:
this doesn't leech stamina it gives it

Reply:
Yes, that is the intent, as stamina gets HIGHER, the user can perform LESS ACTIONS, so Poision adding stamina is intentional.